app/Decsys/Services/PageService.cs

```diff
@@ -16,13 +18,14 @@ namespace Decsys.Services
     /// </summary>
     public class PageService
     {
-        private readonly LiteDatabase _db;
+        private readonly IPageRepository _pages;
```

Comment:
same as components, if we are just getting/updating surveys, and doing all the work in here, then just use `ISurveyRepository` and scrap the `IPageRepository`.